oam/install_scripts/mcs-start-storagemanager.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import configparser
+import sys
+
+config = configparser.ConfigParser()
+config.read('/etc/columnstore/storagemanager.cnf')
+
+storage = config['ObjectStorage']['service']
+region = config['S3']['region']
+bucket = config['S3']['bucket']
+
+if storage.lower() == 's3' and not region.lower() == 'some_region' and not bucket.lower == 'some_bucket':
+    sys.exit(0)
+sys.exit(1)
```

Comment:
For now returning exit code 1. Causes mcs-storagemanager to fail and not startup storagemanager. Wasn't satisfied with this so i tried messing with FailureAction and FailureActionExitStatus in systemd (see mcs-storagemanager.service.in)

Reply:
this has the same typo are the other similar clauses, missed it the first time.

Reply:
have fixed typo